app/src/main/res/values/strings.xml

```diff
@@ -130,6 +130,10 @@
     <string name="create_account_email_format_error">Invalid email format</string>
     <string name="create_account_fields_must_be_filled_error">All fields must be filled</string>
 
+    <!-- Default profile -->
+    <string name="default_display_name">Unnamed hiker</string>
+    <string name="default_email">unnamedhiker@example.com</string>
+
```

Comment:
Warning: Nice job but your forgot to use them 😅

Reply:
Done